Project.toml

```diff
@@ -21,7 +21,7 @@ SparseArrays = "2f01184e-e22b-5df5-ae63-d93ebab69eaf"
 [compat]
 CDDLib = "0.6, 0.7"
 Distributions = "0.19, 0.20, 0.21, 0.22, 0.23, 0.24, 0.25"
-Documenter = "0.23, 0.24, 0.25, 0.26"
+Documenter = "0.23, 0.24, 0.25, 0.26, 0.27"
```

Comment:
should we keep all those previous versions?

Reply:
I think it does not hurt. Sometimes other packages force a downgrade.